Makefile.jenkins

```diff
@@ -46,7 +56,7 @@ endif
 
 # Determine if an end-of-week snapshot is being released and
 # set the "bumped version" accordingly.
-ifeq ($(findstring snapshot, $(RELEASE_TYPE)),snapshot)
+ifeq ($(findstring $(RELEASE_TYPE), snapshot edge),$(RELEASE_TYPE))
```

Comment:
RELEASE_TYPE can be:
* patch
* minor
* major
* snapshot.0
* snapshot.1

And, now, it can also be `edge`... This condition will fail with the snapshot ones :(

(see https://github.com/kiali/kiali/blob/master/deploy/jenkins-ci/Jenkinsfile#L102-L107 and https://github.com/kiali/kiali/blob/master/deploy/jenkins-ci/bin/determine-release-type.sh#L7-L15)

Reply:
You are very right there, I'll follow other approach

Reply:
Updated